ethylene-core/src/main/java/com/github/steanky/ethylene/core/ConfigPrimitive.java

```diff
@@ -126,4 +126,12 @@ public boolean equals(Object obj) {
 
         return false;
     }
+
+    /**
+     * Equivalent to {@code new ConfigPrimitive(null)}.
+     * @return a new ConfigPrimitive containing null
+     */
+    public static @NotNull ConfigPrimitive nullPrimitive() {
+        return new ConfigPrimitive(null);
+    }
```

Comment:
ConfigPrimitive.nullPrimitive()
new ConfigPrimitive(null)

man

Reply:
look i CAN'T do ConfigPrimitive.null() because null is a keyword

Reply:
how about ConfigPrimitive.nil()?

Reply:
ok